docs/examples/using-nonstandard-fresh.md

```diff
@@ -0,0 +1,78 @@
+---
+description: |
+  For cases where the latest release doesn't fit your needs.
+---
+
+Pretend you have a use case where you need to modify your project to use a
+nonstandard version of Fresh. Or you want to use a slightly different
+initialization script. This page has you covered.
+
+## Nonstandard Fresh in `deno.json`
+
+### Latest commit
+
+Let's say you like living life in the fast lane, and want the latest and
+greatest from `main`. How can you modify your project to no longer use the
```

Comment:
We should not encourage people to import directly from `main`, but rather from a specific commit.

Reply:
This also improves the quality of the document as well, since I hadn't covered using a particular commit. And of course the first time I tried doing it (just now) I got the format wrong. Best to have this documented for future users. Thanks.